editor/src/benchmarks.benchmark.ts

```diff
@@ -1,9 +1,13 @@
 import { benchmarkBuildTree } from './core/shared/element-path-tree.benchmark'
 import { benchmarkElementPathFunction } from './core/shared/element-path.benchmark'
 import { benchmarkAttributes } from './core/shared/jsx-attributes.benchmark'
+import { benchmarkOptics } from './core/shared/optics.benchmark'
 import { benchmarkPropertyPathFunction } from './core/shared/property-path.benchmark'
 
+/*
```

Comment:
Delete me?

Reply:
Needs to be kept, because of refactoring the `jsx-attributes.benchmark.ts` benchmark (at least) that causes the benchmarks to fail.